src/pudl/etl/epacems_assets.py

```diff
@@ -26,20 +24,16 @@
 )
 
 import pudl
-from pudl.extract.epacems import EpaCemsPartition
-from pudl.metadata.classes import Resource
-from pudl.metadata.enums import EPACEMS_STATES
+from pudl.extract.epacems import EpaCemsDatastore, EpaCemsPartition
+from pudl.transform.epacems import transform
```

Comment:
A more descriptive alias would help readability for me, like `... import transform as transform_epacems`.